tasks/Task.cpp

```diff
@@ -4,6 +4,12 @@
 
 using namespace apriltags;
 
+#if CV_MAJOR_VERSION >= 4
+const int CV_LINE_TYPE = cv::LINE_AA;
```

Comment:
`cv::LINE_AA` has been available at least since OpenCV3, so we don't need the `#if`-guards (I don't know about OpenCV2, but I don't think we still need to support that version).

Reply:
I guess, in that case we could drop the CV_LINE_TYPE macro in general then and just use cv::LINE_AA directly in the code.

Reply:
Yes, would also be fine for me.